tests/ixia/pfcwd/test_pfcwd_m2o.py

```diff
@@ -7,19 +7,22 @@
     ixia_api_serv_user, ixia_api_serv_passwd, ixia_api, ixia_testbed_config
 from tests.common.ixia.qos_fixtures import prio_dscp_map, all_prio_list,\
     lossless_prio_list, lossy_prio_list
+from tests.common.cisco_data import is_cisco_device
```

Comment:
Don't see any reference of this funcion in the file.

Reply:
Will remove it if not needed.

Reply:
@kevinskwang , I have removed the import.

Reply:
I still can see this code line in the PR